resource_customizations/sparkoperator.k8s.io/SparkApplication/health.lua

```diff
@@ -6,19 +6,10 @@ if obj.status ~= nil then
       health_status.message = "SparkApplication was added, enqueuing it for submission"
       return health_k9sstatus
     end
-    count=0
-    executor_instances = obj.spec.executor.instances
-    for i, executorState in pairs(obj.status.executorState) do
```

Comment:
Is it correct that `obj.status.executorState` has been removed in the newer Spark version? 

In this case can you please support users of older version? E.g. execute old health check if `obj.status.executorState` is present and new if it is missing

Reply:
Hi @alexmt. Newer version of Spark also contains `obj.status.executorState`, but this section is present based on the application state that is checked afterwards. After experimentation with startup of the application I can see why original check was trying to count the executors. Please see the description of PR where I've added the sample data.